### PR TITLE
Map ELB attributes to terraform

### PR DESCRIPTION
**Pull request by justinsb**
Requires moving them under the LoadBalancer awstask, sadly

Depends on #1181

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1182)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
So are we doing tests afterward?

**Comment by justinsb** [comment]
Covered by existing terraform integration tests.

Note `tests/integration/privateweave/kubernetes.tf` - this actually fixes a problem in the TF output.

File: upup/pkg/fi/cloudup/awstasks/load_balancer.go [hunk 1]
```diff
@@ -362,10 +477,11 @@ func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, a, e, chang
 		elbName = e.Name
 	}
 
-	internal := fi.StringValue(e.Scheme) == "internal"
 	tf := &terraformLoadBalancer{
-		Name:     elbName,
-		Internal: internal,
+		Name: elbName,
+	}
+	if fi.StringValue(e.Scheme) == "internal" {
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/awstasks/load_balancer.go]: Is this cast always going to work?

Comment (justinsb) [inline upup/pkg/fi/cloudup/awstasks/load_balancer.go]: FYI that's not a cast, that's a helper function...

This PR is obsolete anyway ... I forgot we'd split out pieces of the mega PR

**Pull request closed without merging**